src/viam/ur/module/ur_arm.cpp

```diff
@@ -49,6 +49,17 @@ pose ur_vector_to_pose(urcl::vector6d_t vec) {
     return pose{position, orientation, theta};
 }
 
+// https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/bff7bf2e2a85c17fa3f88adda241763040596ff1/include/ur_client_library/ur/datatypes.h#L204
+std::string to_robot_type_string(const std::string& model) {
```

Comment:
if we have even more models we might want to change this to a map of some kind. doing so now feels a bit premature